apiary.apib

```diff
@@ -371,6 +371,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `tasks.schedule` endpoint.
+
+- `task_id` filter has been added to `/events.list`.
```

Comment:
This is part of https://github.com/teamleadercrm/api/pull/432 right?

Reply:
B: Indeed. Please let's make it so PRs only contain relevant changes.